### PR TITLE
Add notes for 68

### PR DESCRIPTION
**Pull request by Callisto13**
Check that nothing is missing

### CODE REVIEW

File: docs/release_notes/0.68.0.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- Gracefully delete vpc-cni during cluster deletion (#4244)
+- Drain nodegroups during cluster deletion (#4205)
+- Remove support for 1.16 (#3944)
+- Fix typo in warning message for existing service accounts. (#4233)
+- Update documentation for EKS Connector (#4232)
+
+## Bug Fixes
+
+- Drain nodegroups during cluster deletion (#4205)
+
```
Comment (nikimanoledaki) [inline docs/release_notes/0.68.0.md:18]:
```suggestion
```

Comment (nikimanoledaki) [reply docs/release_notes/0.68.0.md:18]: it was added twice!

**Review by nikimanoledaki — APPROVED** [review]
LGTM! just a small fix

File: docs/release_notes/0.68.0.md (hunk 1 above)
Comment (Callisto13) [reply docs/release_notes/0.68.0.md:18]: twice the value

**Pull request merged**